velox/exec/RowContainer.h

```diff
@@ -204,10 +204,9 @@ class RowContainer {
   // into one word for faster comparison. The bulk allocation is done
   // from 'allocator'. ContainerRowSerde is used for serializing complex
   // type values into the container.
-  /// ''stringAllocator' allows sharing the variable length data arena with
-  /// another RowContainer. this is
-  // needed for spilling where the same aggregates are used for
-  // reading one container and merging into another.
+  // 'stringAllocator' allows sharing the variable length data arena with
```

Comment:
While you are doing this, would you fix all comments for public methods to use /// and comments for internal code to use //?

Reply:
Done.